### PR TITLE
Build storybook on heroku

### PR DESCRIPTION
**Pull request by joelhooks**
Kind of quick and dirty to get it to work. It serves the `storybook_static` folder that is compiled at build time. That is configurable in `static.json`

todo: update README, migrate core styles, and display egghead designs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This repo has 2 purposes:
 - [A component library for npm](https://www.npmjs.com/package/egghead-ui)
 - [A storybook for a living style guide with documentation](https://ui.egghead.io)
 
+### Known Issues
+
+`devDependencies` don't install on heroku. All deps need to be in the `dependencies` object.
```
Comment (joelhooks) [inline README.md:9]: this was annoying, I don't know if there is a solution.

File: package.json [hunk 2]
```diff
@@ -8,13 +8,12 @@
     "build:library": "react-scripts build",
     "build:storybook": "build-storybook",
     "verify": "yarn build:library && yarn build:storybook",
-    "prepublish": "yarn build:library"
+    "prepublish": "yarn build:library",
+    "start": "react-scripts start",
+    "build": "yarn verify"
```
Comment (joelhooks) [inline package.json:13]: this seems kind of crappy. The heroku `create-react-app` buildpack looks for a `build` script. So, there it is. For heroku technically we don't need to build the library.

File: package.json [hunk 3]
```diff
@@ -8,13 +8,12 @@
...
   },
   "devDependencies": {
-    "@kadira/react-storybook-addon-info": "^3.3.0",
-    "@kadira/storybook": "^2.5.2",
-    "react-scripts": "^0.8.4",
-    "react-storybook-addon-props-combinations": "^0.0.1"
+
```
Comment (joelhooks) [inline package.json:16]: there is likely a way to mentally optimize this, but maximizing efficiency isn't really the goal so just using `dependencies` is fine...

File: static.json [hunk 4]
```diff
@@ -0,0 +1 @@
+{ "root": "storybook-static/" }
```
Comment (joelhooks) [inline static.json:1]: this is the folder that gets built and served at https://styleguide.egghead.io

**Review by trevordmiller — COMMENTED** [review]
Looks great :)

**Pull request merged**